dhctl/pkg/kubernetes/actions/deckhouse/delete.go

```diff
@@ -328,10 +259,90 @@ func checkMachinesAPI(kubeCl *client.KubernetesClient) error {
 	return nil
 }
 
+// mcm
+const (
+	MCMGroup        = "machine.sapcloud.io"
+	MCMGroupVersion = "v1alpha1"
+)
+
+func DeleteMCMMachineDeployments(kubeCl *client.KubernetesClient) error {
+	machineDeploymentsSchema := schema.GroupVersionResource{Group: MCMGroup, Version: MCMGroupVersion, Resource: "machinedeployments"}
+	machinesSchema := schema.GroupVersionResource{Group: MCMGroup, Version: MCMGroupVersion, Resource: "machines"}
+
+	return retry.NewLoop("Delete MCM MachineDeployments", 45, 5*time.Second).Run(func() error {
+		allMachines, err := kubeCl.Dynamic().Resource(machinesSchema).Namespace(metav1.NamespaceAll).List(context.TODO(), metav1.ListOptions{})
+		if err != nil {
+			return fmt.Errorf("get machines: %v", err)
+		}
+
+		for _, machine := range allMachines.Items {
+			labels := machine.GetLabels()
+			labels["force-deletion"] = "True"
+			machine.SetLabels(labels)
```

Comment:
How is this annotation used? I do not see any usage across this PR.

Reply:
Needs comment

Reply:
done